examples/spacy/python_env.yaml

```diff
@@ -3,4 +3,6 @@ build_dependencies:
   - pip
 dependencies:
   - mlflow>=1.0
-  - spacy==2.2.3
+  - spacy==2.2.4
```

Comment:
fix error https://github.com/mlflow-automation/mlflow/actions/runs/9633716234/job/26568447318#step:8:203

not very sure why the error happens, but tweaking the micro version works